### ISSUE TITLE
Teams from the same city must not play in the same day

### ISSUE THREAD

**Issue by joaoqalves** [inker/cl-draw#2]
When a city has has two teams in the group phase, they must not play on the same day (meaning, one team in A-D group and the other one in E-H group). Example: Real Madrid and Atlético should not play in the same day

**Comment by inker** [comment]
@joaoqalves, exactly, that's why there are pairings to handle this. See [json/pairings.json](https://github.com/inker/cl-draw/blob/master/json/pairings.json).

But aren't teams normally paired up based on their popularity & subjective strength rather than the proximity factor? I've arbitrarily paired RM with Barça & Atlético with Sevilla. And as far as I know, UEFA never report the real pairings.

**Comment by joaoqalves** [comment]
@inker, there is a proximity factor. From wikipedia:

> There are other restrictions: for example, teams from the same city (e.g., Real Madrid and Atlético Madrid, Chelsea and Arsenal, Manchester City and Manchester United) in general do not play at home on the same matchday (UEFA tries to avoid teams from the same city playing at home on the same day or on consecutive days, due to logistics and crowd control) ...

And I think when a country has only two teams, they should play on different days too. It's a pretty complex system. I could not find a _rule book_, though.

**Comment by inker** [comment]
All right. I guess I'm changing it for now. Thanks for raising the issue.

I'm keeping the issue open in case someone's got any insider information on the subject.
